build.py

```diff
@@ -26,7 +26,8 @@ def get_build_args(target):
                 '--copt', '-fPIC',
             ],
             'win32': [
-                '--define', 'MEDIAPIPE_DISABLE_GPU=1'
+                '--define', 'MEDIAPIPE_DISABLE_GPU=1',
+                '--copt', '/vmg'
```

Comment:
Your solution does work, but I think we should use `TYPED_METHOD_BIND` define from godot-cpp instead.
source: godotengine/godot-cpp/pull/602, godotengine/godot-cpp/issues/1058
(and the defines are moved to `tools/windows.py` later)